helm/vitalam-identity-service/values.yaml

```diff
@@ -0,0 +1,16 @@
+postgresql:
+  enabled: true
+  postgresqlDatabase: vitalam
+  postgresqlUsername: postgres
+  postgresqlPassword: postgres
+config:
+  port: 80
+  logLevel: info
+  dbName: vitalam
+  dbPort: 5432
+  externalPostgresql: null
```

Comment:
Add defaults here if we are not setting them as defaults in `env.js`

Reply:
done